src/plugins/src/volumes/CMakeLists.txt

```
@@ -0,0 +1,3 @@
+set(MTS_PLUGIN_PREFIX "volumes")
+
+add_plugin(grid_spherical grid_spherical.cpp)
```

For consistency with the other Mitsuba plugins, I suggest:
```suggestion
add_plugin(sphericalgridvolume sphericalgrid.cpp)
```
(just like in the docstring)